packages/abstract/src/core/collision/observer.ts

```diff
@@ -67,9 +67,9 @@ export class CollisionObserver<
     collisionDetector?: CollisionDetector
   ) {
     const {registry, dragOperation} = this.manager;
-    const {source, shape, status} = dragOperation;
+    const {source, status} = dragOperation;
 
-    if (!status.initialized || !shape) {
+    if (!status.initialized) {
       return DEFAULT_VALUE;
```

Comment:
It should be collision detection algorithm's responsibility to check shape's availability. File draggable does not have a shape and would be filtered by the original code implementation